src/screens/HomeScreen.js

```diff
@@ -145,7 +145,6 @@ export const HomeScreen = ({ navigation, route }) => {
           refreshTimeKey="publicJsonFile-homeCarousel"
         />
         <Widgets widgetConfigs={widgetConfigs} />
-
```

Comment:
why was this line removed here? it was helpful to separate the several sections.

Reply:
I guess it's a Syntax issue. I will be more careful about this from now on.